libmachine/mcnutils/b2d.go

```diff
@@ -20,12 +19,8 @@ var (
 	GithubApiToken string
 )
 
-const (
-	timeout = time.Second * 5
-)
-
 func defaultTimeout(network, addr string) (net.Conn, error) {
-	return net.DialTimeout(network, addr, timeout)
+	return net.Dial(network, addr)
```

Comment:
Out of curiosity - what was the reasoning for having the timeout initially? Do we have a fallback mechanism in case it wouldn't respond fast enough?